share/brewkit/fix-machos.rb

```diff
@@ -1,8 +1,6 @@
-#!/usr/bin/env ruby
+#!pkgx +gem ruby
```

Comment:
Should be `#!/usr/bin/env -S pkgx +gem ruby`. Shebangs have to be full paths if you are running them directly which with the above change will be true.

Reply:
lol. you beat me by seconds.